jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/resources/org/jbpm/casemgmt/demo/insurance/insurance-rules.drl

```diff
@@ -6,7 +6,7 @@ import org.jbpm.casemgmt.demo.insurance.ClaimReport;
 rule "classify-claim" ruleflow-group "claims"
 
 when 
-    $caseData : CaseData()
+    $caseData : CaseData() from entry-point "insurance-claims.CarInsuranceClaimCase" 
```

Comment:
Yes, it does make it backward incompatible as it introduces complete isolation bwteen the processes